TestPlaybooks/playbook-playbook-CymulateTest.yml

```diff
@@ -0,0 +1,380 @@
+    note: false
+    separatecontext: false
+    task:
+      brand: ""
+      id: f4ed977d-d694-4d0c-8d30-67f0a5e647d8
+      iscommand: true
+      name: fetch_apt_incidents
+      script: '|||fetch_apt_incidents'
+      type: regular
+      version: -1
+    taskid: f4ed977d-d694-4d0c-8d30-67f0a5e647d8
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 600
+        }
+      }
+  "4":
+    id: "4"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "5"
+    note: false
+    separatecontext: false
+    task:
+      brand: ""
+      id: ab71b6f6-1227-42dc-8460-02ba3ec5aefd
+      iscommand: true
+      name: fetch_immediate_threats_incidents
+      script: '|||fetch_immediate_threats_incidents'
+      type: regular
+      version: -1
+    taskid: ab71b6f6-1227-42dc-8460-02ba3ec5aefd
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 800
+        }
+      }
+  "5":
+    id: "5"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "6"
+    note: false
+    separatecontext: false
+    task:
+      brand: ""
+      id: 066c5c99-0635-4aa3-81a5-80789e9640a4
+      iscommand: true
+      name: fetch_phishing_incidents
+      script: '|||fetch_phishing_incidents'
+      type: regular
+      version: -1
+    taskid: 066c5c99-0635-4aa3-81a5-80789e9640a4
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1000
+        }
+      }
+  "6":
+    id: "6"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "7"
+    note: false
+    separatecontext: false
+    task:
+      brand: ""
+      id: 7082bad8-9cfb-45bf-81b3-d7be0d93b1b2
+      iscommand: true
+      name: fetch_waf_incidents
+      script: '|||fetch_waf_incidents'
+      type: regular
+      version: -1
+    taskid: 7082bad8-9cfb-45bf-81b3-d7be0d93b1b2
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1200
+        }
+      }
+  "7":
+    id: "7"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "8"
+    note: false
+    separatecontext: false
+    task:
+      brand: ""
+      id: b9d9132d-3687-476f-8acc-c85827e5a64e
+      iscommand: true
+      name: fetch_hopper_incidents
+      script: '|||fetch_hopper_incidents'
+      type: regular
+      version: -1
+    taskid: b9d9132d-3687-476f-8acc-c85827e5a64e
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1400
+        }
+      }
+  "8":
+    id: "8"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "9"
+    note: false
+    separatecontext: false
+    task:
+      brand: ""
+      id: 3134025d-89bc-4841-8669-8ee43bf155bf
+      iscommand: true
+      name: fetch_dlp_incidents
+      script: '|||fetch_dlp_incidents'
```

Comment:
there are no such commands in your integration
```
fetch_dlp_incidents
fetch_hopper_incidents
fetch_waf_incidents
```

Reply:
update to the right yaml